app/src/androidTest/java/com/android/feedme/test/camera/TextAnalysisTests.kt

```diff
@@ -0,0 +1,153 @@
+package com.android.feedme.test.camera
+
+// import okhttp3.Callback
+
+import com.android.feedme.ml.parseResponse
+import com.android.feedme.model.data.IngredientMetaData
+import com.android.feedme.model.data.MeasureUnit
+import io.mockk.*
+import org.json.JSONObject
+import org.junit.Test
+
+class TextAnalysisTests {
+
+  /*@Test
```

Comment:
We decided to not use this test for the AnalyzedtextforIngredients() function, as it was costing us chat gpt API tokens, for nothing. Keeping this test would have also mean loosing tokens each time the CI would have been run.....